lib/modules/showImages.js

```diff
@@ -260,6 +260,11 @@ module.options = {
 		value: true,
 		description: 'Autoplay inline videos',
 	},
+	keepNativeExpando: {
```

Comment:
Some of the mediaHosts contains something akin to `!elem.classList.contains('title')` in `detect`, but perhaps that can be replaced with an option `isIdenticalToNative`